dependent-usage-analyzer/checkout-dependents.sh

```diff
@@ -0,0 +1,28 @@
+# checkout open edx frontend repos
+mkdir .projects
+(
+  cd .projects &&
+  git clone git@github.com:edx/frontend-app-account.git --depth 1
```

Comment:
Would this approach handle finding Paragon usage in edx-platform? Is it worth trying to find usage there?

Reply:
It may, I can try it.

Reply:
With some additional resilience added around parsing javascript code with babel this works for edx-platform. It's now been added to the checkout script above.